javascript-practice/src/js/views/userView.js

```diff
@@ -1,19 +1,59 @@
+import { validatePassword } from '../helpers/validation'
+import { ERROR_MESSAGE } from '../constants/message'
+
+export const {
+  REQUIRED_FIELD_EMAIL,
+  REQUIRED_FIELD_PASSWORD,
+  INVALID_EMAIL,
+  INVALID_PASSWORD,
+} = ERROR_MESSAGE
+
 export default class UserView {
   constructor() {
-    this.loginFormEl = document.getElementById('login-form');
+    this.loginFormEl = document.getElementById('login-form')
+    this.emailFormEl = document.getElementById('email')
+    this.passwordFormEl = document.getElementById('password')
+    this.emailErrorEl = document.getElementById('email-error')
+    this.passwordErrorEl = document.getElementById('password-error')
   }
 
   bindFormLogin = (submitLogin) => {
-    this.loginFormEl.addEventListener("submit", e => {
-      e.preventDefault();
-      const valueEmail = document.getElementById('email').value;
-      const valuePassword = document.getElementById('password').value;
-      
-      submitLogin(valueEmail, valuePassword); 
-    });
+    this.loginFormEl.addEventListener('submit', (e) => {
+      e.preventDefault()
```

Comment:
Check this style pls: https://github.com/airbnb/javascript?tab=readme-ov-file#semicolons